src/components/JobModalFields.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+export default (props) => {
+
+    let styleRight = {
+        float: "right",
+        clear: "both"
+    };
+    let styleLeft = {
+        float: "left",
+        clear: "both"
+    };
+    let divStyle = {
+        marginTop: "5%",
+        marginBottom: "15%"
+    };
+    let container = {
+        marginLeft: "10px",
+        marginRight: "10px",
+        marginTop: "10%",
+        marginBottom: "10%"
+    }
+    return (
+      
+        <div style={{ top: '50%', left: '50%', transform: `translate(-50%,-50%)` }} className={props.className}>
```

Comment:
here we're using `inline styles`, i.e. `{{ top: '50%'` and mixing them with `CSS-In-JS` above.  We shouldn't do that - one or the other